### PR TITLE
updating readme for preview2

### PR DESCRIPTION
**Pull request by sayedihashimi**
@balachir let's review when you come in. We need to update FWLinks for downloads.

### CODE REVIEW

File: known-issues.md [hunk 1]
```diff
@@ -259,28 +259,39 @@ Note: There is a known issue with Visual Studio 2015 Update 3 RC, as a result of
 
 ### Missing SDK
 
-If you open an RC2 project (or RTW) and the corresponding .NET Core SDK is missing you'll get a dialog box similar to the following.
+If you open a Preview 2 (1.0.1) project and the corresponding .NET Core SDK is missing you'll get a dialog box similar to the following.
```
Comment (balachir) [inline known-issues.md]: How about just "a .NET Core project"? The dialog will appear for any case where the corresponding .NET Core SDK is missing.

File: known-issues.md [hunk 2]
```diff
@@ -259,28 +259,39 @@ Note: There is a known issue with Visual Studio 2015 Update 3 RC, as a result of
...
 
-![error box missing sdk](./images/missing-sdk-rtm.png)
+![error box missing sdk](./images/missing-sdk-preview2.png)
 
-To workaround this you can install the correct version of the .NET SDK. You can use the links below.
+#### Previw 2 (1.0.1) SDK download links
```
Comment (balachir) [inline known-issues.md]: .NET Core 1.0.1 SDK download links

File: known-issues.md [hunk 3]
```diff
@@ -259,28 +259,39 @@ Note: There is a known issue with Visual Studio 2015 Update 3 RC, as a result of
...
 | Platform    | Link          |
 |-------------|---------------|
-| Windows x64 | https://go.microsoft.com/fwlink/?LinkID=798398 |
-| Windows x86 | https://go.microsoft.com/fwlink/?LinkID=798399 |
+| Windows x64 | ***TODO: Update with actual FWLink*** |
+| Windows x86 | ***TODO: Update with actual FWLink*** |
+
 
-#### RTM SDK download links
+#### RTM (1.0.1) SDK download links
```
Comment (balachir) [inline known-issues.md]: .NET Core 1.0.0 SDK Preview 2 download links

**Comment by sayedihashimi** [comment]
@balachir FYI updated based on your comments

File: known-issues.md [hunk 4]
```diff
@@ -259,28 +259,39 @@ Note: There is a known issue with Visual Studio 2015 Update 3 RC, as a result of
...
 | Platform    | Link          |
 |-------------|---------------|
-| Windows x64 | https://dotnetcli.blob.core.windows.net/dotnet/preview/Installers/Latest/dotnet-dev-win-x64.latest.exe |
-| Windows x86 | https://dotnetcli.blob.core.windows.net/dotnet/preview/Installers/Latest/dotnet-dev-win-x86.latest.exe |
+| Windows x64 | ***TODO: Update with actual FWLink*** |
+| Windows x86 | ***TODO: Update with actual FWLink*** |
 
 For more info see https://github.com/dotnet/cli/#installers-and-binaries
 
+#### RC2 (1.0.0) SDK download links
```
Comment (balachir) [inline known-issues.md]: .NET Core 1.0.0 RC2 SDK 1.0.0-preview1-002702 download links

File: known-issues.md [hunk 5]
```diff
@@ -259,28 +259,39 @@ Note: There is a known issue with Visual Studio 2015 Update 3 RC, as a result of
 
 ### Missing SDK
```
Comment (balachir) [inline known-issues.md]: Should we move this section to the top of the known issues page (ie. under latest release) ?

File: release-notes.md [hunk 6]
```diff
@@ -38,6 +38,28 @@ Fixes in this update include the following:
 * Fix for a bug that sometimes caused crashes in Visual Studio 2015 Update 3 (`CLR_EXCEPTION_System.NullReferenceException_80004003_Microsoft.VisualStudio.Composition.dll!Microsoft.VisualStudio.Composition.ExportProvider+PartLifecycleTracker.ReportPartiallyInitializedImport`)
 * Fix for a [bug](https://github.com/aspnet/Tooling/issues/647) that sometimes caused .xproj to be repeatedly built with the message `(project) will be compiled because inputs were modified`.
 
+## Release notes for .NET Core 1.0.0 SDK Preview 2 – VS 2015 Tooling Preview 2
```
Comment (balachir) [inline release-notes.md]: .NET Core 1.0.1 - VS 2015 Tooling Preview 2

File: release-notes.md [hunk 7]
```diff
@@ -38,6 +38,28 @@ Fixes in this update include the following:
...
+
+This release contains the following.
+
+### RTM tooling for ASP.NET (ASP.NET, not including ASP.NET Core)
+This release contains RTM quality support for ASP.NET. Including many bug fixes from previous releases.
```
Comment (balachir) [inline release-notes.md]: Should we remove the "Including many bug fixes .." part? We didn't change tooling for ASP.NET in this release

File: release-notes.md [hunk 8]
```diff
@@ -38,6 +38,28 @@ Fixes in this update include the following:
...
+
+### Preview 2 tooling for .NET Core and ASP.NET Core
+
+This release contains Preview 1 tooling for .NET Core and ASP.NET Core. We have also fixed a number of issues from previous releases.
```
Comment (balachir) [inline release-notes.md]: Remove this section as well. "We have also fixed a number of issues from previous releases" is not correct

**Comment by sayedihashimi** [comment]
@balachir just updated it with all your comments

File: release-notes.md (hunk 6 above)
Comment (balachir) [inline release-notes.md]: Also, these release notes are for the latest release, so it should be at the top

**Pull request merged**